src/sec_adapter_svp.c

```diff
@@ -39,12 +39,20 @@ Sec_Result SecOpaqueBuffer_Malloc(SEC_SIZE bufLength, Sec_OpaqueBufferHandle** h
         return SEC_RESULT_FAILURE;
     }
 
-    sa_status status = sa_svp_buffer_alloc(&(*handle)->svp_buffer, bufLength);
+    sa_status status = sa_svp_memory_alloc(&(*handle)->svp_memory, bufLength);
     if (status != SA_STATUS_OK) {
         free(*handle);
```

Comment:
Do you need to log an error here and on line 51?

Reply:
Done